gnmi/subscribe/subscribe.go

```diff
@@ -0,0 +1,587 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package subscribe implements the gnmi.proto Subscribe service API.
+// This is a forked version of github.com/openconfig/gnmi's subscribe package.
+package subscribe
```

Comment:
This would help me with figuring out what code I should be reviewing too :-)

Reply:
Added start and ending banners. The change is simply to avoid sending the gNMI updates by adapting the current `Subscribe` behaviour, such that it only populates the queue with the updates, and that's it. I'll have to look at it more closely to figure out a forking strategy because I'm basically adding a functionality that's not needed by gNMI.